ansible/group_vars/wakuv2-prod.yml

```diff
@@ -32,6 +32,9 @@ nim_waku_websockify_ssl_dir: '{{ nim_waku_websocket_ssl_dir }}'
 nim_waku_websockify_ssl_cert: '{{ nim_waku_websocket_ssl_cert }}'
 nim_waku_websockify_ssl_key: '{{ nim_waku_websocket_ssl_key }}'
 
+# Waku rln-relay parameters
+nim_waku_rln_relay_content_topic: '/toy-chat/2/luzhou/proto'
```

Comment:
What's the point of this setting it for prod if `rln-relay` is not enabled for it?

Reply:
I think with. my recent change and checking whether rln-relay is enabled in the protocols set, then I can safely remove this variable from waku v2 prod variables.